Julia/cp/isPowerOf2.jl

```diff
@@ -0,0 +1,26 @@
+#= Given a number N check if it is a power of 2 in O(1). We will be 
+using bitwise operator to get the calculation.=#
+
+## Function
+
+function isPowerOfTwo(n)
+    return (n & (n - 1) == 0)
+end
+
+## Input
+
+n = readline()
```

Comment:
Prompt user to enter the number.

Reply:
@atarax665 I haven't prompted the user to enter the number as that is the template of all the other codes present in the Julia folder and other languages folder as well. You could have a look at all the existing codes present in the database. This style is similar to them.

Reply:
> @atarax665 I haven't prompted the user to enter the number as that is the template of all the other codes present in the Julia folder and other languages folder as well. You could have a look at all the existing codes present in the database. This style is similar to them.

We have recently added Julia to the project, soon we will be coming up with a new template that is more interactive and beginner-friendly.

Reply:
@atarax665 will take care of that in the newer Julia codes I add, I did the changes to make it as per you requested, do review it.